Tests/MapboxMapsTests/Foundation/DelegatingViewAnnotationPositionsUpdateListenerTests.swift

```diff
@@ -0,0 +1,33 @@
+import XCTest
+@testable import MapboxMaps
+
+final class DelegatingViewAnnotationPositionsUpdateListenerTests: XCTestCase {
+
+    private class Receiver: DelegatingViewAnnotationPositionsUpdateListenerDelegate {
+
+        var positions: [ViewAnnotationPositionDescriptor] = []
+
+        internal func onViewAnnotationPositionsUpdate(forPositions positions: [ViewAnnotationPositionDescriptor]) {
+            self.positions = positions
+        }
+
+    }
```

Comment:
We typically put these in separate files and name them according to the pattern `Mock<ProtocolName>`

We'd also use `Stub` to capture the arguments instead of a simple property.

Reply:
Mock and test source files are organized like this:

For mocks, if the protocol `C` is located at `/Sources/A/B/C.swift`, then the corresponding mock would be placed at `/Tests/A/B/Mocks/MockC.swift`

For tests, if the entity `Z` is located at `/Sources/X/Y/Z.swift`, then the corresponding the test would be place at `/Tests/X/Y/ZTests.swift`

Reply:
cc: @OdNairy FYI